src/site/antora/antora.yml

```diff
@@ -0,0 +1,50 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+#  ██     ██  █████  ██████  ███    ██ ██ ███    ██  ██████  ██
+#  ██     ██ ██   ██ ██   ██ ████   ██ ██ ████   ██ ██       ██
+#  ██  █  ██ ███████ ██████  ██ ██  ██ ██ ██ ██  ██ ██   ███ ██
+#  ██ ███ ██ ██   ██ ██   ██ ██  ██ ██ ██ ██  ██ ██ ██    ██
+#   ███ ███  ██   ██ ██   ██ ██   ████ ██ ██   ████  ██████  ██
+#
+#  THIS FILE IS A STUB!
+#
+#  The actual `antora.yml` that is used in `target/generated-site/antora` is auto-generated from `antora.tmpl.yml`.
+#  Auto-generation happens during `pre-site` phase of Maven.
+#  Hence, you must always
+#
+#  1. Edit `antora.tmpl.yml`
+#  2. Edit `antora.yml` to match the fields in `antora.tmpl.yml` and fill Maven property placeholders with dummy values
+#
+
+name: ROOT
+title: Home
+version: ~
+start_page: index.adoc
+asciidoc:
+  attributes:
+    project-github-url: "https://github.com/apache/logging-log4j-tools"
+    project-version: "1.2.3"
+    project-name: "Log4j Tools"
+    project-id: "log4j-tools"
+    java-target-version: "8"
+    java-compiler-version: "[17,18)"
```

Comment:
You might want to fix the mentions of Log4j Tools here.

Reply:
check